homeassistant/components/vesync/diagnostics.py

```diff
@@ -0,0 +1,141 @@
+"""Diagnostics support for VeSync."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyvesync import VeSync
+
+from homeassistant.components.diagnostics import REDACTED
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.helpers.device_registry import DeviceEntry
+
+from .common import VeSyncBaseDevice
+from .const import DOMAIN, VS_MANAGER
+
+KEYS_TO_REDACT = {"manager"}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    manager: VeSync = hass.data[DOMAIN][VS_MANAGER]
+
+    data = {}
+    data.update(
+        {
+            DOMAIN: {
+                "account_id": manager.account_id,
+                "bulb_count": len(manager.bulbs),
+                "fan_count": len(manager.fans),
+                "outlets_count": len(manager.outlets),
+                "switch_count": len(manager.switches),
+                "timezone": manager.time_zone,
+            }
+        }
+    )
+
+    data.update(
```

Comment:
Home Assistant will automatically add this config entry info

Reply:
that block doesn't get auto-added, it's intent was to place config_entry values in the output.  That block came from wherever I sourced the original code and didn't appear to add value.  Removed the data.